src/components/ScholarshipAmountField.jsx

```diff
@@ -23,65 +24,80 @@ function ScholarshipAmountField(props) {
     minAmount,
     maxAmount,
     onTypeChange,
-    onMinChange,
-    onMaxChange,
-    updateMinMax,
+    updateAmount,
   } = props;
 
+  function amountTextField(label, value, onChange) {
+    return (
+      <TextField
+        className={classes.formControl}
+        label={label}
+        type="number"
+        InputLabelProps={{ shrink: true }}
+        InputProps={{ inputProps: { min: 0, step: 50 } }}
+        disabled={
+          amountType !== AmountType.Range && amountType !== AmountType.Fixed
+        }
+        value={value}
+        onChange={onChange}
+      />
+    );
+  }
+
   function displayAmountFields() {
-    if (amountType === AmountType.Fixed) {
-      return (
-        <TextField
-          label="Amount"
-          type="number"
-          InputLabelProps={{ shrink: true }}
-          fullWidth
-          InputProps={{ inputProps: { min: 0, step: 50 } }}
-          value={minAmount}
-          onChange={(e) => updateMinMax(parseInt(e.target.value, 10) || '')}
-        />
-      );
-    }
-    if (amountType === AmountType.Range) {
-      return ['minAmount', 'maxAmount'].map((field) => (
-        <TextField
-          key={field}
-          label={field}
-          type="number"
-          InputLabelProps={{ shrink: true }}
-          fullWidth
-          InputProps={{ inputProps: { min: 0, step: 50 } }}
-          value={field === 'minAmount' ? minAmount : maxAmount}
-          onChange={field === 'minAmount' ? onMinChange : onMaxChange}
-        />
-      ));
+    switch (amountType) {
+      case AmountType.Range:
+        return (
+          <>
+            {amountTextField('Minimum Amount', minAmount, (e) =>
+              updateAmount('minAmount', parseInt(e.target.value, 10) || '')
+            )}
+            {amountTextField('Maximum Amount', maxAmount, (e) =>
+              updateAmount('maxAmount', parseInt(e.target.value, 10) || '')
+            )}
+          </>
+        );
+      default:
+        return amountTextField('Amount', minAmount, (e) =>
+          updateAmount(
+            'minAmount',
+            parseInt(e.target.value, 10) || '',
+            'maxAmount'
+          )
+        );
     }
-    return <div />;
   }
```

Comment:
Instead of this you could just have them show up next to the relevant radio button options. E.g. something like
```
( ) Unknown  
( ) Fixed Amount: $[Amount]
( ) Range: $[Min Amount] to $ [Max Amount]
( ) Full Tuition
```

And just disable the ones for options that aren't selected based on `amountType`